include/arch/arm/arch/32/mode/machine_pl2.h

```diff
@@ -121,10 +121,23 @@ static inline void invalidateHypTLB(void)
 static inline paddr_t PURE addressTranslateS1(vptr_t vaddr)
 {
     uint32_t ipa0, ipa1;
+    UNUSED uint32_t saved_ipa0, saved_ipa1;
+    if (config_set(CONFIG_ARM_HYPERVISOR_SUPPORT)) {
```

Comment:
This section of this file is already within an `#ifdef CONFIG_ARM_HYPERVISOR_SUPPORT` and there's another definition of `addressTranslateS1()` for non-hyp configs. So because this function is specifically for the hyp config these if guards shouldn't be added here.

Reply:
Thanks, right I will remove them ;)

Reply:
addressed.